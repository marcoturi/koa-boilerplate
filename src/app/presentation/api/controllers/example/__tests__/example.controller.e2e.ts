import createTestServer, {IServerTest} from "../../../../../../../test/utils/createTestServer";


describe('Given common routes', () => {
    let serverTest: IServerTest;
    beforeEach(async () => {
        serverTest = await createTestServer();
    });

    describe('GET /api/example', () => {
        test('should return example', async () => {
            const req = await serverTest.request
                .get('/api/v1/example')
                .expect(200)
                .expect('Content-Type', /json/);

            const roles = req.body;
            expect(roles).toMatchSnapshot();
        });
    });
});
