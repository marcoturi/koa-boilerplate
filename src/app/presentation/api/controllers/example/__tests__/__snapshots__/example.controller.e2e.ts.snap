// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given common routes GET /api/example should return example 1`] = `
{
  "result": "example",
}
`;
